ha/resource/sspl

```diff
@@ -115,22 +115,45 @@ stateful_check_state() {
     return 7
 }
 
+start_sspl_service() {
+        systemctl start sspl-ll
+        echo "START (master): $?"
+        while true
+        do
+            status=`systemctl is-active sspl-ll`
+            if [[ $status == "active" ]]; then
+                return 0
+            elif [[ $status == "failed" ]]; then
```

Comment:
what about "unknown" status?
How should that be handled?
@ajay-paratmandali, have you checked that in dynamicFid resource agent?

Reply:
@ajaysrivas What is the expected behavior when the current service state is "unknown"?  Do we need to wait till the service starts or returns immediately?

Reply:
Not sure. This is the reason, I asked question :-)
Probably we should check if it remains "unknown" for some time (5 sec) then return 1.

Reply:
Yes, It's good to wait for 5 sec. Did the changes in the latest patch.

Reply:
I meant, continuously for 5 sec. So the count should be set to zero in other if conditions. I guess, 5 sec is fine for a stable status, so this is okay too.